perlin.py

```diff
@@ -92,10 +94,10 @@ def apply_network_noise(net: sumolib.net.Net, xml: ElementTree, scale: float = 0
         eid = edge.getID()
         if eid not in known_streets:
             # This edge is missing a street entry. Find population and industry for this edge
-            population = get_population_number(edge=edge, base=POPULATION_BASE, scale=scale, octaves=octaves,
-                                               centre=centre, radius=radius)
-            industry = get_population_number(edge=edge, base=INDUSTRY_BASE, scale=scale, octaves=octaves,
-                                             centre=centre, radius=radius)
+            population = get_population_number(edge=edge, base=POPULATION_BASE, scale=noise_scale, octaves=3,
+                                               centre=centre, radius=radius, centre_weight=0.8)
+            industry = get_population_number(edge=edge, base=INDUSTRY_BASE, scale=noise_scale, octaves=3,
+                                             centre=centre, radius=radius, centre_weight=0.1)
```

Comment:
We should probably allow for users to override the default weights

Reply:
Yes. Agree.